src/components/datepicker/FullNavigationBar.js

```diff
@@ -0,0 +1,100 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Box } from '../box';
+import { Button, ButtonGroup } from '../button';
+import Section from '../section';
+import { Heading2 } from '../typography';
+import {
+  IconArrowLeftMediumOutline,
+  IconArrowRightMediumOutline,
+  IconArrowLeftSmallOutline,
+  IconArrowRightSmallOutline,
+} from '@teamleader/ui-icons';
+
+const PADDINGS = {
+  small: 3,
+  medium: 4,
+  large: 5,
+};
+
+class FullNavigationBar extends PureComponent {
+  handlePreviousClick = () => {
+    // Function to trigger the next button of our calendar
+    const { onPreviousClick: onDayPickerPreviousClick } = this.props;
+    onDayPickerPreviousClick();
+
+    // Function passed from our parent component to update our state
+    this.props.onClickPrevious();
```

Comment:
All the onNextClick, onPreviousClick, onClickNext, ... are not required according to the propTypes, but there is no check on whether they were provided or not.

Reply:
Good remark, they should all be required.

Reply:
We don't have a proper YearPicker in our library. We have a DatePicker with the option to skip 12 months but this picker misses some crucial items. Right now we have in mind that this component could be used to replace the year calendar in my account for example.